### ISSUE TITLE
Xserver breaks when not in foreground

### ISSUE THREAD

**Issue by deb75** [pelya/xserver-xsdl#84]
Hi,

I use xserver-xsdl (latest) with linux-deploy.

I noticed that as soon as I put the xserver app in background, it merely stops.

When I try to come back to it using the previous apps android menu, it seems to start all over again.

Isn't it possible to make it persistent ? Like most apps do ? Or am I doing something wrong ?

Regards

**Comment by mark-orion** [comment]
I have the same problem. The x-connection breaks the moment the app is in the background or the lockscreen kicks in. This makes the app almost unusable with linux-deploy.

**Comment by deb75** [comment]
Hi,

Does lockscreen really breaks the x-connection ?

For what I have tested, if the X-server is in foreground and that you lock screen or put your device in standby mode, it recovers when you wake up the device or unlocks the screen, but provided it was in foreground.

Tested on a sm-p900 with (still) unofficial lineageos system, linux-deploy and xserver-sdl installed a few months ago, they should still be relevant.

I agree that if x-connection would break like you say at locking screen, it would be far too annoying.
